test/sys/test_pthread.rs

```diff
@@ -3,5 +3,5 @@ use nix::sys::pthread::*;
 #[test]
 fn test_pthread_self() {
     let tid = pthread_self();
-    assert!(tid > 0);
+    assert!(tid != 0);
```

Comment:
Do I just remove the test leaving the `test_pthread.rs` file for a future test to be inserted later or should I completely remove the file ?

Concerning the validity of the test. Maybe we have to wait `pthread_create` is defined in nix, so that we could make a test where we spin a thread and test that `pthread_self` returns different `pthread_t` in the different threads.
And we shouldn't even test for an "invalid" return value as `pthread_self` is not meant to fail, man page says it always succeed.